file.rb

```diff
@@ -0,0 +1,14 @@
+module Utils
+  class Socket
+    def self.from_uri(uri)
+      host, port = URI.parse(uri)
+      new(host, port)
+    end
+
+    def initialize(host, port)
```

Comment:
Utils::Socket::initialize has increased in size the last 3 times it has been modified - [5, 4, 3]